### PR TITLE
Features/sma 27 Add Audit and Rename files for better navigation

### PR DESCRIPTION
**Pull request by filmakarov**
# Summary

Add Account Recovery Audit by Zellic and Rename other audit files for better navigation

Related Issue: SMA 27

## Change Type
- [x] Documentation Update

### CODE REVIEW

File: contracts/smart-account/interfaces/modules/IOrgUserAdminModule.sol [hunk 1]
```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+/**
```
Comment (ankurdubey521) [inline contracts/smart-account/interfaces/modules/IOrgUserAdminModule.sol]: Empty natspec

File: contracts/smart-account/modules/OrgUserAdminModule.sol [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import {IAuthorizationModule} from "../interfaces/IAuthorizationModule.sol";
+import {ISignatureValidator} from "../interfaces/ISignatureValidator.sol";
+
+/**
+ * @author Fil Makarov - <filipp.makarov@biconomy.io>
+ */
+
+contract OrgUserAdminModule {
+    // destination => smartAccount => selector => isAllowed
+    mapping(address => mapping(address => mapping(bytes4 => bool)))
```
Comment (ankurdubey521) [inline contracts/smart-account/modules/OrgUserAdminModule.sol]: With recent versions of solidity you can write it directly as 
`mapping(address destination => mapping(address smartAccount => mapping(bytes4 selector => bool isAllowed)))`

File: contracts/smart-account/modules/OrgUserAdminModule.sol [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+import {ECDSA} from "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import {IOrgUserAdminModule} from "../interfaces/modules/IOrgUserAdminModule.sol";
+import {IAuthorizationModule} from "../interfaces/IAuthorizationModule.sol";
+import {ISignatureValidator} from "../interfaces/ISignatureValidator.sol";
+
+/**
+ * @author Fil Makarov - <filipp.makarov@biconomy.io>
+ */
+
+contract OrgUserAdminModule {
```
Comment (ankurdubey521) [inline contracts/smart-account/modules/OrgUserAdminModule.sol]: Should it implement `ISignatureValidator` ?

File: contracts/smart-account/modules/OrgUserAdminModule.sol [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+            uint256 offset = uint256(bytes32(userOp.callData[4 + 64:4 + 96]));
+            uint256 length = uint256(
+                bytes32(userOp.callData[4 + offset:4 + offset + 32])
+            );
+            //we expect data to be the `IERC20.transfer(address, uint256)` calldata
+            data = userOp.callData[4 + offset + 32:4 + offset + 32 + length];
+        }
+
+        // doesn't pass
+        //bool res = _allowedDestinations1[destination][userOp.sender][bytes4(data[0:4])];
```
Comment (ankurdubey521) [inline contracts/smart-account/modules/OrgUserAdminModule.sol]: do we need this comment?

**Review by ankurdubey521 — APPROVED** [review]

**Review by Aboudjem — APPROVED** [review]

**Pull request merged**